.circleci/config.yml

```diff
@@ -8,10 +8,10 @@ workflows:
       - architect/push-to-app-catalog:
           context: "architect"
           # executor: "app-build-suite" # uncomment this if you want automatic metadata generation and helm chart linting
-          name: "package and push {APP-NAME} chart"
+          name: "package and push rbac chart"
           app_catalog: "giantswarm-playground-catalog"
```

Comment:
is the playground catalog the right one?

Reply:
I don't think so, but I don't know which one it will be. I will add it to official one as it is only configuration